keyboards/doio/kb04/readme.md

```diff
@@ -0,0 +1,27 @@
+# doio/kb04
+
+![doio/kb04](https://cdn.shopify.com/s/files/1/0463/5980/6101/products/4_f40396cc-ea72-47a5-887f-bbeeb4023b11_1100x.png?v=1654787686)
```

Comment:
This image should be rehosted on Imgur.

Reply:
Added in https://github.com/qmk/qmk_firmware/pull/22754/commits/545dca3f9e648d561353514d0b0f196b4dbfdf67.